projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -625,6 +625,8 @@ postgresql:
 ## See https://github.com/cockroachdb/helm-charts/tree/master/cockroachdb
 cockroachdb:
    enabled: true
+   tls:
+    enabled: false
```

Comment:
Can you explain in more detail why this needs to be disabled and what the impact of this is? Not sure I understood from the PR desc

Reply:
Sure, on the newer versions of cockroach ssl is enabled by default. 
We need it disabled by default which was the old behaviour.